.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+# # Use the latest 2.1 version of CircleCI pipeline process engine.
```

Comment:
I can kinda understand this but I'm excited to talk more about it so I can understand it better!